src/azure-cli/azure/cli/command_modules/sql/aaz/latest/sql/mi/dns_alias/_update.py

```diff
@@ -0,0 +1,359 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "sql mi dns-alias update",
+)
+class Update(AAZCommand):
+    """Updates an Azure SQL Managed Instance DNS alias.
+
+    Updates the managed instance DNS alias by creating or removing the DNS record. If the --has-dns-record parameter is passed, the DNS record will be created, otherwise it will be removed (if it exists).
+
+    :example: Updates a managed instance DNS alias by creating the DNS record
+        az sql mi dns-alias update -g <resourceGroupName> --mi <managedInstanceName> --name <dnsAliasName> --dns-record
+
+    :example: Updates a managed instance DNS alias, with the given resource id, by removing the DNS record (not specifying the --dns-record parameter)
+        az sql mi dns-alias update --ids /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/<resourceGroupName>/providers/Microsoft.Sql/managedInstances/<managedInstanceName>/dnsAliases/<dnsAliasName>
+    """
+
+    _aaz_info = {
+        "version": "2021-11-01-preview",
```

Comment:
api-version is preview, recommand to release those commands as preivew

Reply:
SQL APIs are usually released only in preview versions but are treated as stable. I think that we don't need to release those commands as preview